eng/AfterSolutionBuild.targets

```diff
@@ -15,7 +15,7 @@
     <RepoTasks.RemoveSharedFrameworkDependencies Condition="@(_BuildOutput->Count()) != 0"
       Files="@(_BuildOutput)"
       FrameworkOnlyPackages="@(AspNetCoreAppReference)"
-      SharedFrameworkTargetFramework="net$(AspNetCoreMajorVersion).$(AspNetCoreMinorVersion)" />
+      SharedFrameworkTargetFramework="net5.0" />
```

Comment:
Use of `$(AspNetCoreMajorVersion)` or `$(AspNetCoreMajorMinorVersion)` was only occasionally correct. Since the concepts aren't related, we shouldn't change this back in the future